### DISCUSSION TITLE
Initial start without Screen or KBD

### DISCUSSION THREAD

**Discussion by svScoots** [bareboat-necessities/lysmarine_gen#186] [Q&A] [answered]
Is there a way to start from first load without a screen and keyboard?  I will be ultimately using via VNC, which is supported, but I can't find a way to get the initial setup done.  Is there a way to SSH or VNC in, without first setting it up with a keyboard and screen? I'm out cruising and have neither on board and I'd like to try BBN.

Eric

**Comment by mgrouch** [comment, marked as answer]
It will still boot with SSH available. So can SSH and set up VNC from terminal.
It might be even be accessible via VNC if you boot it up without screen or keyboard. Give it 10-15 mins time (it needs to do one reboot on it's own) before
trying accessing it with VNC.

**Reply by svScoots** [reply]
I may not have waited that long.  I'll give it another try now I know it should work.
Waited a much longer time and it worked, the VNC desktop didn't display properly but I was able to SSH in and fix things.
